### DISCUSSION TITLE
Jakarta 10 compatible Bouncy castle jar

### DISCUSSION THREAD

**Discussion by khusi1251** [bcgit/bc-java#1557] [Q&A]
Hi All,

we are upgrading our product for which we are using java 17 and Jakarta 10 and we need Jakarta 10 compatible bouncy castle jars.  do we have Jakarta compatible bouncy castle jars available, if yes than where can we get those jars?

Thanks,
Khushboo

**Comment by khusi1251** [comment]
Any idea on asked query?

**Comment by khusi1251** [comment]
Any idea on asked query?

**Comment by DidierLoiseau** [comment]
There is bcjmail available, see #832

**Comment by prathamlashkari** [comment]
Hi Khushboo,

Yes, there are Jakarta-compatible Bouncy Castle jars available. You can find them on the Bouncy Castle official website or through Maven Central. Just search for the Jakarta version you need.

Hope this helps!
